packages/frontend/src/components/accounts/SetupImplicit.js

```diff
@@ -96,7 +97,7 @@ class SetupImplicit extends Component {
         this.setState({ creatingAccount: true });
         await Mixpanel.withTracking("CA Create account from implicit",
             async () => {
-                await dispatch(createAccountFromImplicit(newAccountId, implicitAccountId, recoveryMethod));
+                await dispatch(createAccountFromImplicit({ accountId: newAccountId, implicitAccountId, recoveryMethod }));
```

Comment:
This might be a dumb question (my redux is a little rusty) but why do some async thunks get wrapped in `dispatch` and some are just `await`'d directly? If calling `dispatch` on them is a no-op can we always just `await` them directly?

Reply:
@andy-haynes class components use `mapDispatchToProps` and therefore don't need the explicit `dispatch` when called. We're generally moving away from class components in favor of functional components using explicit `dispatch`:)